locales/en-US/addon.properties

```diff
@@ -18,7 +18,8 @@ add_to_queue= Add to queue
 clear= Clear
 history= History
 play_queue= Play Queue
-track_added_notification= Track added to queue
+track_added_notification= Item added to queue
```

Comment:
You need to use a new ID here, otherwise tools will maintain the existing translation
This is valid also for add-ons https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings